mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MessageMapperTest.java

```diff
@@ -365,7 +365,7 @@ public void findFirstUnseenMessageUidShouldReturnUid2WhenUid2isSeen() throws Mai
         messageMapper.updateFlags(benwaInboxMailbox, new FlagsUpdateCalculator(new Flags(Flags.Flag.SEEN), FlagsUpdateMode.REPLACE), MessageRange.one(message5.getUid()));
         assertThat(messageMapper.findFirstUnseenMessageUid(benwaInboxMailbox)).isEqualTo(message2.getUid());
     }
-
+/*
```

Comment:
Ouups?

Reply:
No oupss, the `expungedMarkedForDeletion` method has been removed, so I need to refactor those tests to take the new methods into account, but I wanted to be sure that it was compiling and that I didn't break anything else at that stage... :)